lib/ibanity.rb

```diff
@@ -57,6 +66,10 @@ def api_schema
       @api_schema ||= client.get(uri: client.base_uri)["links"]
     end
 
+    def isabel_connect_schema
+      @isabel_connect_schema ||= client.get(uri: "#{client.base_uri}/isabel-connect")["links"]
```

Comment:
rename `api_schema` to `xs2a_api_schema` and use  `client.get(uri: "#{client.base_uri}/xs2a")["links"]` then.
For the sandbox we can continue to use xs2a_api_schema until we implement it as well

Reply:
also, rename to `isabel_connect_api_schema`

Reply:
I've renamed the method to xs2a_api_schema but we'll have to reroute in our backend to allow /xs2a to work; currently it maps to /application/xs2a in xs2a

Reply:
I've renamed the method to xs2a_api_schema but we'll have to changing the routing to allow /xs2a to work; currently it maps to /application/xs2a in xs2a